backend/grant/task/jobs.py

```diff
@@ -126,8 +126,50 @@ def process_task(task):
             })
 
 
+class PruneDraft:
+    JOB_TYPE = 4
+    PRUNE_TIME = 259200  # 72 hours in seconds
+
+    def __init__(self, proposal):
+        self.proposal = proposal
+
+    def blobify(self):
+        return {
+            "proposal_id": self.proposal.id,
+        }
+
+    def make_task(self):
+        from .models import Task
+
+        task = Task(
+            job_type=self.JOB_TYPE,
+            blob=self.blobify(),
+            execute_after=self.proposal.date_created + timedelta(seconds=self.PRUNE_TIME),
+        )
+        db.session.add(task)
+        db.session.commit()
+
+    @staticmethod
+    def process_task(task):
```

Comment:
I wonder if the task will re-run indefinitely, or does the task itself get deleted after it's ran? Reminder to self to double check how tasks work.

Reply:
I was curious about this as well! Each task is ran in the API view and is marked as `completed  = True` if it runs successfully. Here's the code block for reference:

https://github.com/grant-project/zcash-grant-system/blob/494303883ae44c3e3a250a8f92ee416ef7f2b2d8/backend/grant/task/views.py#L13-L25